TokenExtractor/AuthorizationHeaderTokenExtractor.php

```diff
@@ -17,11 +17,18 @@ class AuthorizationHeaderTokenExtractor implements TokenExtractorInterface
     protected $prefix;
 
     /**
-     * @param string $prefix
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @param string $pr
+     * @param string $name
      */
-    public function __construct($prefix)
+    public function __construct($prefix, $name)
```

Comment:
It would be safer to set a default value of 'Autorization' for the $name argument, to prevent any BC break.